app/js/components/queryBuilder/queryParamSelector/diseaseTypeInstructions/diseaseType_instructions.tpl.html

```diff
@@ -8,9 +8,9 @@ <h1 class="col-lg-12" >BUILD A CLASSIFICATION ALGORITHM BY:</h1>
 					<h2><b>Searching for Samples by Disease Type</b></h2>
 				</li>
 				<li>
-					<h2><b>Selecting Samples by Disease Type to add to your Disease Type Set by Clicking</b></h2>
+					<h2><b>Selecting Samples by Disease Type to add to your Disease Type Set by Clicking or Selecting all Samples from All Diseases by Clicking the "Select All" Button</b></h2>
```

Comment:
You may want to cut this line into two.

Reply:
What do you think should be a good standard character length for html?

Reply:
`adage-server` project is using 80, which is based on google's style guide:
https://github.com/google/eslint-config-google